rfcs/text/0000-multi-data-collection-dataset-processing.md

```diff
@@ -0,0 +1,233 @@
+### DCP PR:
+
+***Leave this blank until the RFC is approved** then the **Author(s)** must create a link between the assigned RFC number and this pull request in the format:*
+
+`[dcp-community/rfc#](https://github.com/HumanCellAtlas/dcp-community/pull/<PR#>)`
+
+# Processing Datasets that Span Multiple Data Collection Runs
+
+## Summary
+This RFC proposes a design solution to allow datasets that span multiple data collection runs to be processed in unison as per the scientific requirements. The proposal generalizes to multiple data type modalities, but also provides specific details for 10X V2 3’ single-cell data as an example and to address an immediate need for a data processing solution.
+
+
+## Author(s)
+[Nick Barkas](mailto:barkasn@broadinstitute.org),
+[Mark Diekhans](mailto:markd@ucsc.edu)
+
+## Shepherd
+-[Nick Barkas](mailto:barkasn@broadinstitute.org)
+
+## Motivation
+Correct data processing requires the initiation of data processing pipelines with all the appropriate input data. The current mechanism of bundle notifications is not effective when data spans multiple bundles because there is no clear set of rules to identify the bundles that need to be co-processed.  Limiting the downstream process to the scope of a single bundle requires the submitter to define the scope of the processing without any foreknowledge of future processes needs.
```

Comment:
Submitters bear no responsibility for how data is bundled internally. There is a set of rules for how data is bundled that the submitter has no control over.

Reply:
This should be reworded.  Where are the rules documented?  I would assume they still trigger bundling based on the experiment design (per assay??), so there is a connection from what the submitter creates and how ingest creates bundles.  We need to keep this rules-based and not put more burden on the user.

Reply:
@justincc Is there a good place for documentation on how the exporter created bundles right now other than the code?

Reply:
https://docs.google.com/document/d/1cGfolHMGEe1IKkXSBS75SAdSri9Np_oW6Z4S9-Ax1QY/edit. Not complete so updates welcome